ci/dace.yml

```
@@ -21,12 +17,11 @@ test_model_stencils_aarch64:
 .test_model_datatests:
   stage: test
   script:
-    - pip install dace==$DACE_VERSION
-    - tox -r -e run_model_tests -c model/ -- --backend=$BACKEND $COMPONENT  --verbose
+    - nox -s "test_model_datatest-3.10(subpackage='$COMPONENT')" -- --backend=$BACKEND
   parallel:
     matrix:
-    - COMPONENT: [atmosphere/diffusion/tests/diffusion_tests]
-      BACKEND: [dace_cpu_noopt]
+    - COMPONENT: [atmosphere/advection, atmosphere/diffusion, atmosphere/dycore]
```

We have to remove the advection tests, DaCe test execution is hanging there.

```suggestion
    - COMPONENT: [atmosphere/diffusion, atmosphere/dycore]
```

---

Or we can leave it hang, but we should shorten the `SLURM_TIMEOUT`, currently set to 6 hours:
https://github.com/C2SM/icon4py/blob/3068bd783bc5531a999d6ab9c1c66d203d9117dd/ci/base.yml#L69